stacks/SiriGeneratorStack.ts

```diff
@@ -8,14 +8,37 @@ export function SiriGeneratorStack({ stack }: StackContext) {
     const { disruptionsJsonBucket } = use(SiteStack);
 
     const siriSXBucket = new Bucket(stack, "SiriSXBucket", {
-        name: `cdd-siri-sx-${stack.stage}`,
+        name: `cdd-siri-sx-generator-${stack.stage}`,
+        cdk: {
+            bucket: {
+                encryption: BucketEncryption.S3_MANAGED,
+                versioned: true,
+                blockPublicAccess: {
+                    blockPublicAcls: true,
+                    blockPublicPolicy: true,
+                    ignorePublicAcls: true,
+                    restrictPublicBuckets: true,
+                },
+            },
+        },
     });
 
     const siriSXUnvalidatedBucket = new Bucket(stack, "SiriSXUnvalidatedBucket", {
-        name: `cdd-siri-sx-unvalidated-${stack.stage}`,
+        name: `cdd-siri-sx-validator-${stack.stage}`,
```

Comment:
This should stay as `cdd-siri-sx-unvalidated-${stack.stage}`, you may have updated the bucket names rather than add in the function names

Reply:
done